src/pages/resources/recipes/file-operation.md

```diff
@@ -0,0 +1,317 @@
+# File operations
+
+To perform any file operations including read, write, create, and delete, UXP provides a couple of options. But before we look at the APIs, let's get acquainted with a few concepts.
+
+## Concepts
+
+### Sandbox and other locations
+UXP, by default, only allows access to certain locations in the user's file system. These locations are referred to as the sandbox. 
+
+<InlineAlert variant="info" slots="header, text1, text2"/>
+
+Scripts and plugins
+
+**In plugins**, a sandbox is typically the plugin's folder, plus a temporary and data folder. The plugin's location is read-only. Whereas, the temporary and data folders are provided to you to store transitory information. Note that the data stored here can get accidentally erased and should not be considered permanent.
+
+**In scripts**, lorem ipsum. // TODO What about sandbox locations in scripts?
```

Comment:
@kareemmyk

Reply:
Sandbox for scripts is only ```temporary folder```.

Reply:
@kareemmyk  Does this mean the scheme 'plugin-temp:/' will work for InDesign UXP scripts?